frontend/src/styles/material.sass

```diff
@@ -6,3 +6,18 @@
  *.mat-tab-group.mat-primary .mat-ink-bar
  *    background-color: red
  */
+
+snack-bar-container
+    &.error-snack-bar 
+        background: #a90707
+        text-align: center
+    &.success-snack-bar 
+        background: #07be53
+        text-align: center
+    &.warning-snack-bar 
+        background: #d68408
```

Comment:
See previous comment

Reply:
I need to merge new version of development branch, because now we have separate sass file for colors